src/test/java/com/google/devtools/build/lib/query2/cquery/ConfiguredTargetQuerySemanticsTest.java

```diff
@@ -120,9 +119,9 @@ public void testLabelFunction_getCorrectlyConfiguredDeps() throws Exception {
     setUpLabelsFunctionTests();
 
     // Test that this retrieves the correctly configured version(s) of the dep(s).
-    ConfiguredTarget patchDep =
+    KeyedConfiguredTarget patchDep =
```

Comment:
Can we add a test case for the situation that caused #11993 ?

Reply:
Good point. Added to //src/test/shell/bazel:toolchain_transition_test